src/modules/app/containers/LanguageFragment.js

```diff
@@ -0,0 +1,110 @@
+import { Fragment } from 'redux-little-router'
```

Comment:
This file will need tests now because we need to test the withFetcher calls. the fetch(...) call needs to get mocked then.

Reply:
yep sure, as I said tests are missing and this will require some more work. I wouldn't include this in our release.